content/javascript/concepts/window-prompt/window-prompt.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'prompt()'
+Description: 'Displays a dialog with an optional message asking users to input some text.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`prompt()`** method is serviced globally by the `window` object provided by the user's browser. It displays a dialog with an optional message asking users to input some text.
```

Comment:
function*

Reply:
Since we're talking within the scope of window object, we'll refer to it as such